core/app/[locale]/(default)/account/[tab]/_components/product-snippet.tsx

```diff
@@ -0,0 +1,149 @@
+import { useFormatter } from 'next-intl';
+
+import { graphql, ResultOf } from '~/client/graphql';
+import { BcImage } from '~/components/bc-image';
+import { PricingFragment } from '~/components/product-card';
+import { cn } from '~/lib/utils';
+import { Link } from '~/navigation';
+
+export const ProductCardFragment = graphql(
```

Comment:
Why not import `ProdutCardFragment` from `/components/product-card`? If it has less nodes, I would rename.